CHANGELOG_UNRELEASED.md

```diff
@@ -26,6 +26,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - in `poly.v`
   + lemmas `coef_prod_XsubC`, `coefPn_prod_XsubC`, `coef0_prod_XsubC`
 
+- in `ssral.v`
```

Comment:
```suggestion
- in `ssralg.v`
```

Reply:
This does not look resolved. Please do not click "resolve" until the fix is pushed. If you push a commit fixing it, it will resolve itself automatically.